### DISCUSSION TITLE
Choose Type Won't Change (Docker)

### DISCUSSION THREAD

**Discussion by pbuss2** [tteck/Proxmox#1177] [Help Needed] [answered]
Hi,

I am trying to install a privileged version of docker, however I can't get the choose type to go to () 0 Privileged. I am using shell on my PC so that could be the problem. I have tried typing 0 at this screen or moving the cursor down and hitting enter. Am I missing something obvious?

![image](https://user-images.githubusercontent.com/123527608/225515908-17017c5a-3a72-4d0e-963d-920e80842dd1.png)

Thanks!

Paul

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/issues/394#issuecomment-1201338725
